gem_rspec/test_case_spec.rb

```diff
@@ -13,7 +13,7 @@
     end
 
     context "with a template file" do
-      let!(:template) { SmashingDocs.config { |c| c.template_file = 'spec/fake_template.md.erb' } }
+      let!(:template) { SmashingDocs.config { |c| c.template_file = 'gem_rspec/fake_template.md.erb' } }
```

Comment:
This is just used within the gem for testing, right? Just making sure it's not the default template.